server/src/main/java/org/elasticsearch/transport/TransportService.java

```diff
@@ -82,9 +82,6 @@ public class TransportService extends AbstractLifecycleComponent
     private static final boolean SERVERLESS_TRANSPORT_FEATURE_FLAG;
     static {
         final boolean serverlessFlag = Booleans.parseBoolean(System.getProperty(SERVERLESS_TRANSPORT_SYSTEM_PROPERTY), false);
```

Comment:
nit: this entire static block could be removed and the value set directly on the static member

Reply:
Done.